docs/scratch_orgs.rst

```diff
@@ -1,38 +1,172 @@
+
+1. `Install the Salesforce CLI <https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_install_cli.htm>`_
+2. `Enable Dev Hub in Your Org <https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_enable_devhub.htm>`_
+3. `Connect SFDX to Your Dev Hub Org <https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_auth_web_flow.htm>`_ (be sure to use the ``--setdefaultdevhubusername`` option).
+
+If you already have the ``sfdx`` command installed, have connected to your Dev Hub, and have set the ``defaultdevhubusername`` config setting (use ``sfdx force:config:list`` to verify), you're ready to start using CumulusCI with Salesforce CLI.
+
+You can learn more about the Salesforce CLI at https://developer.salesforce.com/platform/dx.
+
+
+Predefined Orgs
 ---------------
 
-Create a scratch org
+CumulusCI comes with five org predefined org configurations. Every org's keychain starts with these configurations available and ready to be turned into a live scratch org:
+
+* ``dev`` is for development workflows and uses the ``orgs/dev.json`` configuration file. It has a seven-day lifespan.
+* ``qa`` is for testing workflows. ``qa`` and ``dev`` are the same out of the box, but can be customized to suit the needs of the project. It has a seven-day lifespan.
+* ``feature`` is for use in continuous integration and uses the ``orgs/dev.json`` configuration file. It has a one-day lifespan.
+* ``beta`` is for use in continuous integration or hands-on testing. It uses the ``orgs/beta.json`` configuration. It has a one-day lifespan.
+* ``release`` is for use in continuous integration, hands-on testing, or demo workflow. It uses the ``orgs/release.json`` configuration. It has a one-day lifespan.
+
+
+Create a Scratch Org
 --------------------
 
-Implicit creation of scratch orgs
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+To create a scratch org from a configuration, simply use it as the target of a CumulusCI command, task, or flow. CumulusCI automatically initializes orgs when they're first used. To create a scratch org from the ``dev`` configuration and review information about the created org, run
+
+.. code-block:: console
+
+    $ cci org info dev
+
+Once the org is created, it's associated with the name ``dev`` in the CumulusCI keychain and can be used with other commands until it expires. Once an org expires or is removed, its associated configuration is left in place, and can be recreated whenever needed.
+
+You can create new org configurations that inherit their configuration from a built-in org. For example, to create a new org name that uses the same configuration as type ``dev``, you can use the command ::
```

Comment:
I'd be inclined to put this in a separate section further down after covering other more basic operations.